src/InvertedIndex.java

```diff
@@ -4,22 +4,29 @@
 
 public class InvertedIndex {
     public static HashMap<String, ArrayList<String>> wordToDocumentMap = new HashMap<>();
+
+    public static String[] wordSplitter(Map.Entry<String, String> file) {
+        String content = file.getValue();
+        return content.split("\\s");
+    }
+
+    public static void makeWordToDocumentMap(String[] words, String fileName) {
```

Comment:
According to the method functionality, "addWordsToDocumentMap" makes more sense for its name.